src/style.css

```diff
@@ -0,0 +1,205 @@
+:root {
+  --primary: rgb(71, 69, 69);
+  --secondary: #ccc;
+  --border: rgb(221, 219, 219);
+  --background: rgb(243, 243, 243);
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.flex-center {
+  align-items: center;
+}
+
+body {
+  width: 100%;
+  min-height: 100vh;
+  background-color: var(--background);
+  font-size: 1.4rem;
+}
+
+i {
+  cursor: pointer;
+  color: var(--secondary);
+}
+
+section {
+  background-color: #fff;
+  width: 50%;
+  min-width: 300px;
+  box-shadow: 0 3px 10px rgb(0 0 0 / 20%);
+  border-radius: 6px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.header {
+  border-bottom: 1px solid var(--border);
+  padding: 1.2rem 1.5rem;
+  color: var(--primary);
+  justify-content: space-between;
+}
+
+form button {
+  background-color: transparent;
+  border: none;
+  padding: 0.6rem;
+}
+
+form button i {
+  transform: rotate(90deg);
+  font-size: 1rem;
+  font-weight: 400;
+}
+
+.header i {
+  transform: rotate(90deg);
+  font-size: 18px;
+}
+
+form > div {
+  border-bottom: 1px solid var(--border);
+  padding: 0.7rem 1.5rem;
+  justify-content: space-between;
+  width: 100%;
+}
+
+form input {
+  border: none;
+  outline: none;
+  min-width: 250px;
+  padding: 0.8rem 0;
+  flex: 1;
+}
+
+form input,
+form input::-webkit-input-placeholder {
+  font-size: 1.2rem;
+  font-family: 'Roboto', sans-serif;
+  font-style: italic;
+  font-weight: 100;
+  color: rgb(131, 130, 130);
+}
+
+.tasks-list {
+  min-width: 100%;
+}
+
+.tasks-list li {
+  width: 100%;
+  list-style: none;
+  border-bottom: 1px solid var(--border);
+  padding: 0.7rem 1.2rem;
+}
+
+.tasks-list li label {
+  margin-left: 6px;
+  font-size: inherit;
+  color: var(--primary);
+  cursor: pointer;
+}
+
+/* .tasks-list li label.compeleted {
+  text-decoration: line-through;
+  font-style: italic;
+  color: var(--secondary);
+} */
```

Comment:
# Hello [@BenMukebo ],
## You have done a great project!
However, there are few  changes you have to make:
- Remove all unused lines of code in order to have a lean codebase

Reply:
Hi @Whoistolu, Thanks for your feedback 👏🏻